.gitattributes

```diff
@@ -0,0 +1,2 @@
+rails/bin/bundle eol=lf
+rails/bin/rails eol=lf
```

Comment:
Does it support wildcards? I think all those files would have the same problem if the extra CR was introduced. How about this:

```suggestion
rails/bin/* eol=lf
```